platform.bible-extension/README.md

```
@@ -152,6 +152,38 @@ To build the extension once:
+
+3. Example uses of this network service are found in [add-word.web-view.tsx](src/web-views/add-word.web-view.tsx) and [find-word.web-view.tsx](src/web-views/find-word.web-view.tsx). Note the following key elements:
+
+- With types `NetworkObject` imported from `'@papi/core'` and `IEntryService` imported from `'lexicon'`:
+
+```
+const [lexiconService, setLexiconService] = useState<NetworkObject<IEntryService> | undefined>();
+```
+
+- With `networkObjects` imported from `'@papi/frontend'`, a `useEffect` to fetch the network service via
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)